bamoe-kie-flyway-migrations/kogito-apps/data-index/data-index-storage/data-index-storage-jpa/src/main/resources/kie-flyway/db/data-index/microsoft-sql-server/V1.45.0.9__create_data_index_storage_sqlserver_bamoe920.sql

```diff
@@ -0,0 +1,262 @@
+    process_id character varying(255) NOT NULL,
+    addon character varying(255) NOT NULL,
+    CONSTRAINT processes_addons_pkey PRIMARY KEY (process_id, addon),
+    CONSTRAINT fk_processes_addons_processes FOREIGN KEY (process_id) REFERENCES processes(id) ON DELETE CASCADE
+);
+
+CREATE TABLE processes_roles (
+    process_id character varying(255) NOT NULL,
+    role character varying(255) NOT NULL,
+    CONSTRAINT processes_roles_pkey PRIMARY KEY (process_id, role),
+    CONSTRAINT fk_processes_roles_processes FOREIGN KEY (process_id) REFERENCES processes(id) ON DELETE CASCADE
+);
+
+CREATE TABLE tasks (
+    id character varying(255) NOT NULL,
+    actual_owner character varying(255),
+    completed datetimeoffset(6),
+    description character varying(255),
+    endpoint character varying(255),
+    inputs nvarchar(max),
+    last_update datetimeoffset(6),
+    name character varying(255),
+    outputs nvarchar(max),
+    priority character varying(255),
+    process_id character varying(255),
+    process_instance_id character varying(255),
+    reference_name character varying(255),
+    root_process_id character varying(255),
+    root_process_instance_id character varying(255),
+    started datetimeoffset(6),
+    state character varying(255),
+    external_reference_id character varying(4000),
+    sla_due_date datetimeoffset(6),
+    CONSTRAINT tasks_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE tasks_admin_groups (
+    task_id character varying(255) NOT NULL,
+    group_id character varying(255) NOT NULL,
+    CONSTRAINT tasks_admin_groups_pkey PRIMARY KEY (task_id, group_id),
+    CONSTRAINT fk_tasks_admin_groups_tasks FOREIGN KEY (task_id) REFERENCES tasks(id) ON DELETE CASCADE
+);
+
+CREATE TABLE tasks_admin_users (
+    task_id character varying(255) NOT NULL,
+    user_id character varying(255) NOT NULL,
+    CONSTRAINT tasks_admin_users_pkey PRIMARY KEY (task_id, user_id),
+    CONSTRAINT fk_tasks_admin_users_tasks FOREIGN KEY (task_id) REFERENCES tasks(id) ON DELETE CASCADE
+);
+
+CREATE TABLE tasks_excluded_users (
+    task_id character varying(255) NOT NULL,
+    user_id character varying(255) NOT NULL,
+    CONSTRAINT tasks_excluded_users_pkey PRIMARY KEY (task_id, user_id),
+    CONSTRAINT fk_tasks_excluded_users_tasks FOREIGN KEY (task_id) REFERENCES tasks(id) ON DELETE CASCADE
+);
+
+CREATE TABLE tasks_potential_groups (
+    task_id character varying(255) NOT NULL,
+    group_id character varying(255) NOT NULL,
+    CONSTRAINT tasks_potential_groups_pkey PRIMARY KEY (task_id, group_id),
+    CONSTRAINT fk_tasks_potential_groups_tasks FOREIGN KEY (task_id) REFERENCES tasks(id) ON DELETE CASCADE
+);
+
+CREATE TABLE tasks_potential_users (
+    task_id character varying(255) NOT NULL,
+    user_id character varying(255) NOT NULL,
+    CONSTRAINT tasks_potential_users_pkey PRIMARY KEY (task_id, user_id),
+    ADD CONSTRAINT fk_tasks_potential_users_tasks FOREIGN KEY (task_id) REFERENCES tasks(id) ON DELETE CASCADE

```

Comment:
```suggestion
   CONSTRAINT fk_tasks_potential_users_tasks FOREIGN KEY (task_id) REFERENCES tasks(id) ON DELETE CASCADE
```

The `ADD` makes the sql fail...